### PR TITLE
Got cnes working with new version of sonarqube

### PR DESCRIPTION
**Pull request by Xvier**
#Proposed changes
* The Cnes plugin doesnt work with the sonarqube 8.0+ api, they changed the way to find projects. I Added a piece of code in the javascript part to check what version of sonarqube you are running so that it will call the right api.

### CODE REVIEW

**Comment by Xvier** [comment]
The first travis ci build didnt work because of the sonarqube none variable not working.
When sonarqube runs it looks for localhost:9000 instead of the sonarcloud. I dont know if you guys removed the sonarcloud key in your project but thats why all normal merge request will fail.

Also java8 isnt supported anymore in travisci so if you look in the travis.yml i commented the sonarqube none and java8 version

**Comment by Sancretor** [comment]
Hi @Xvier !

Thanks a lot for your contribution, we really appreciate it !
I also have to apologize for the (very) long delay... we had a hard time finding time to manage this project.

Regarding the Travis-CI failure, it's a know problem that is partially resolved on the dev branch.
I don't why there is no sonarcloud key in the travis config file, we'll correct that.
Thanks for the tip too !

File: .travis.yml [hunk 1]
```diff
@@ -10,11 +10,11 @@ addons:
     organization: "lequal"
 
 jdk:
-  - oraclejdk8
```
Comment (Sancretor) [inline .travis.yml]: Could you please revert your modifications on the `.travis.yml` file ?
This issue regarding Java8 was resolved in the dev branch, and we'll fix our sonarcloud key issue on this branch too.

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>fr.cnes.sonar</groupId>
     <artifactId>cnesreport</artifactId>
-    <version>3.1.0</version>
+    <version>3.1.8</version>
```
Comment (Sancretor) [inline pom.xml]: Could you please revert this change in `pom.xml` too ?
We'd like to keep control on the version of the plugin for now.

File: scripts/run-sonar.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (Sancretor) [inline scripts/run-sonar.sh]: Could you please explain why you created this script ? For the Travis CI process ?
If I'm correct, it is never called...

**Comment by Xvier** [comment]
I will do the changes this week. I just got a baby so its also hectic at my part :)

**Comment by Sancretor** [comment]
Congratulations !

**Comment by Sancretor** [comment]
Hi @Xvier !

As we would like to publish a new version of CNES Report, with your contribution, we will merge this PR and deal with the conflicts by ourselves.
Take care of your little one, and thanks again for your contribution !
